collections/brandedUserProfile.js

```diff
@@ -26,47 +28,59 @@ BrandedUserSchema = new SimpleSchema({
     appCompleteDate: fields.date(o),
     decisionDate: fields.date(o),
     applicationType: fields.string(o)
-  }), optional: true},
+  }),
+  optional: true},
   created: fields.date(),
   collegeName: fields.string(o),
   college: fields.string(o),
   collegeId: fields.string({optional: false}),
   control: fields.bool(o),
   dob: fields.date(o),
   email: fields.string(o),
-  enrollmentId: fields.string(o), //pantherId for gsu
+  enrollmentId: fields.string(o), // pantherId for gsu
```

Comment:
Is this field GSU only? Wondering if it's different from CRM id or whatever we use for that typically.

Reply:
I am not super clear on the meaning of `enrollmentId` these days; I think we tend to map unique id columns that we aren't using as the offical unique id of the school to this field in imports. Is that right=, @AnthonyReid99 ? In any case, this PR isn't touching that.

Reply:
Oh right i see that this is just a whitespace change. My appologies!

Reply:
@djrochford @jeremysimms it isn't really used for anything now. just a common field that schools supply us in data uploads